cypress/e2e/buy-shirt.cy.ts

```diff
@@ -18,6 +18,11 @@ const paymentStepPage = new PaymentStepPage();
 
 describe("Buy a t-shirt", () => {
   it("then should be bought a t-shirt", () => {
```

Comment:
also, here, you can initialize the page objects in a before hook, like we do in the cypress/e2e/dresses-list.cy.ts